CODEOWNERS

```diff
@@ -1,6 +1,9 @@
 # Use owners-snuba by default
 * @getsentry/owners-snuba
 
+# rust_snuba
+* @getsentry/owners-snuba @getsentry/owners-processing
```

Comment:
I don't think this is right, this makes them codeowners for the entire repo

Reply:
oops, fixed